<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGrid</class>
 <widget class="QDialog" name="DialogGrid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>175</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_columns">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_betapromile">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_rows">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>10</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Columns</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rows</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Beta (in promile)</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinBox_rows</tabstop>
  <tabstop>spinBox_columns</tabstop>
  <tabstop>spinBox_betapromile</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogGrid</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogGrid</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
